src/main/java/edu/greenblitz/tobyDetermined/RobotMap.java

```diff
@@ -62,12 +64,26 @@ public static class Vision {
 		public static double standardDeviationVisionAngle = 0.1;
 		public static int selectedTagId = 1;
 		public static final int[] portNumbers = {5800,5801,5802,5803,5804,5805};
-		public static final Pose3d apriltagLocation = new Pose3d(new Translation3d(5, 96.4, 0), new Rotation3d(0, 0, Math.PI));
-		static List<AprilTag> apriltags = new ArrayList<>(5) ;
+		public static final Pose3d redApriltagLocationId1 = new Pose3d(new Translation3d(15.513558, 1.071626, 0), new Rotation3d(0, 0, Math.PI));
```

Comment:
Cant we read this from json file?
If not - create new Const file, this is not const of the robot